packages/react-server/docs/client-transitions.md

```diff
@@ -0,0 +1,131 @@
+# Client Transitions
+
+React-server provides blazing fast server-side render of web pages.  This is
+important for landing experience and for web crawlers, but it's only part of
+the picture!  The great strength of `react-server` is that the _same code_
+runs in the browser.  Once you've loaded a page, you _could_ ask the server to
+render the next page for you, but why bother?  Why not just fetch the _data_
+for the page and render it yourself?
+
+That's where _client transitions_ come in.  Navigation between `react-server`
+pages can be done without ever hitting the server.  It's easy.  Pages don't
+need to do anything tricky to make this work.  It's part of the framework!
+
+## The `<Link>`
+
+How easy is it?
+
+```javascript
+const {Link} = require("react-server");
+
+const MyPageLink = () => <Link path="/my-page">My page!</Link>
+```
+
+That easy!  The `<Link>` component exported by `react-server` produces `<a>`
+tags with click handlers that automatically turn normal navigation into client
+transitions.  Middle-clicks, command-clicks, etc, all still behave as
+expected, but a normal left-click to navigate becomes a light-weight
+transition within the browser.
+
+## The `ClientRequest`
+
+Want to manage your own `<a>` tags and click handlers?  Want to navigate based
+on some other action?
+
+```javascript
+const {
+	ClientRequest
+	History,
+	getCurrentRequestContext,
+} = require("react-server");
+
+getCurrentRequestContext().navigate(
+	new ClientRequest(this.props.path),
+	History.events.PUSHSTATE
+);
+```
```

Comment:
We've got to make this easier.  I want to be able to initiate a client transition with a single import.

Something as simple as:

```javascript
const {navigateTo} = require("react-server");

navigateTo(path, {options});
```